identity/src/javaSharedMain/kotlin/com/android/identity/crypto/CryptoJvm.kt

```
@@ -329,7 +330,7 @@ actual object Crypto {
                     signature.toDerEncoded()
                 }
             }
-            Signature.getInstance(signatureAlgorithm).run {
+            Signature.getInstance(signatureAlgorithm, BouncyCastleProvider.PROVIDER_NAME).run {
```

This should be avoided since it might cause long pauses while the provider is loaded. Better to fail early so the app can initialize at app startup instead of when the pause is very noticable (for example during credential presentation).

---

OK, I am only adding it for brainpool curves now where it does not work on Android otherwise.